nshlib/nsh_parse.c

```diff
@@ -174,13 +161,6 @@ static void nsh_alist_free(FAR struct nsh_vtbl_s *vtbl,
                            FAR struct nsh_alist_s *alist);
 #endif
 
-#ifndef CONFIG_NSH_DISABLEBG
-static void nsh_releaseargs(struct cmdarg_s *arg);
```

Comment:
remove np_bg process from nsh_builtin.c, nsh_fileapps. and nsh_timcmds.c too.

Reply:
nsh_builtin & nsh_fileapps will handle bg independently. to minum the influence,did not change the nsh_builtin & nsh_fileapps code.